oidc-controller/api/templates/verified_credentials.html

```diff
@@ -0,0 +1,301 @@
+            <div class="button-content">
+              <div class="message">Check Status</div>
+              <div class="icon"></div>
+            </div>
+          </button>
+          <button id="refresh-button" class="qr-button" title="Refresh QR Code">
+            <div class="button-content">
+              <div class="message">QR code scanned</div>
+              <div class="fake-button">
+                <div class="icon">
+                  {{add_asset("refresh.svg")}}
+                </div>
+                <div class="description">Refresh QR code</div>
+              </div>
+            </div>
+          </button>
+          {{add_asset("dashed-border.svg")}}
+          <img
+            src="data:image/jpeg;base64,{{image_contents}}"
+            alt="{{image_contents}}"
+            width="300px"
+            height="300px"
+          />
+        </div>
+
+        <div>
+          <b>Don't have a digital wallet?</b>
+        </div>
+        <a
+          title="Download BC Wallet"
+          href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-wallet"
+          >Download BC Wallet</a
+        >
+      </div>
+    </div>
+  </body>
+  <script>
+    let state = "intro"; // intro, success, failure
+
+    const toggleState = (state) => {
+      const intro = document.querySelector(".intro");
+      const success = document.querySelector(".success");
+      const failure = document.querySelector(".failure");
+      const refreshButton = document.getElementById("refresh-button");
+      const introButton = document.getElementById("intro-button");
+
+      switch (state) {
+        case "intro":
+          // Header elements
+          intro.style.display = "grid";
+          success.style.display = "none";
+          failure.style.display = "none";
+
+          // Button elements
+          refreshButton.style.display = "none";
+          introButton.style.display = "flex";
+          break;
+        case "success":
+          // Header elements
+          intro.style.display = "none";
+          success.style.display = "grid";
+          failure.style.display = "none";
+
+          // Button elements
+          refreshButton.style.display = "flex";
+          introButton.style.display = "none";
+
+          setTimeout(() => {
+            window.location.replace("{{callback_url}}", {method: 'POST'})
+          }, 3000);
+          break;
+        case "failure":
+          // Header elements
+          intro.style.display = "none";
+          success.style.display = "none";
+          failure.style.display = "grid";
+
+          // Button elements
+          refreshButton.style.display = "flex";
+          introButton.style.display = "none";
+          break;
+      }
+    };
+
+    document.getElementById("refresh-button").addEventListener("click", () => {
+      toggleState("intro");
+      state = "intro";
+    });
+
+    document.getElementById("intro-button").addEventListener("click", () => {
```

Comment:
Could we have this set to be executed every 2000ms, so that the user will not need to click the QR code to move on, when the proof-request is verified?